README.md

```diff
@@ -287,10 +296,10 @@ scor <- StrCorr(predictions, Y[-trainIdx])
 ``` r
 scor
 #> $s
-#> [1] 0.8451333
+#> [1] 0.845
```

Comment:
It may be a RcppZiggurat seed issue. I don't think I knew about that when I made the first version of the README.Rmd file.  Something to check would be to add a `RcppZiggurat::zsetseed(1234)` call before each `RerF` call on both branches and compare.

Reply:
Seems like we should be setting that RcppZiggurat::zsetseed() equal to the seed passed into the RerF function if it does make a difference

Reply:
The following code (in Linux) on staging and the commit which you first added the Rmd file (ee9a43c83af2626d3d331523e002fec6e170a6f7) produce different results (when specifying the RcppZiggurat seed).

```
library(rerf)
set.seed(24)
nsamp <- 30 ## number of training samples per species
trainIdx <- vapply(list(1:50, 51:100, 101:150), sample, outer(1,1:nsamp), size = nsamp)
X <- as.matrix(iris[,1:4])
Y <- iris[[5L]]
RcppZiggurat::zsetseed(1234)
forest <- RerF(X[trainIdx, ], Y[trainIdx], num.cores = 1L, seed = 1L)
predictions <- Predict(X[-trainIdx, ], forest, num.cores = 1L, aggregate.output = FALSE)
scor <- StrCorr(predictions, Y[-trainIdx])
scor
```

Staging:
```
scor
$s
[1] 0.845

$rho
[1] 0.4186858
```

ee9a43c83af2626d3d331523e002fec6e170a6f7

```
scor
$s
[1] 0.8451333

$rho
[1] 0.4141002
```

On ee9a43c83af2626d3d331523e002fec6e170a6f7, omitting the `RcppZiggurat::zsetseed(1234)` line, we return to the values in the `Readme.md` file.

```
scor
$s
[1] 0.8451333

$rho
[1] 0.4141002
```

In staging, omitting the ziggurat seed does not change the result.

@MrAE, I tried searching through the changes since your initial Rmd commit but couldn't find anything major regarding `seed`.  Perhaps you can take a look with fresh eyes? https://github.com/neurodata/lumberjack/compare/ee9a43c83af2626d3d331523e002fec6e170a6f7...staging#diff-c85d4d520961c2a3f307505ae988f04c